docs/release.md

```diff
@@ -0,0 +1,132 @@
+</p>
+
+The FHIR packaging specification is still under development and does not have
+a fully fleshed out release process, but we can still use the versioning,
+package naming, etc. guidelines in the specification as a basis for the
+Kids First release specification.
+
+## Versioning
+We will follow the FHIR NPM package specification's versioning scheme which
+uses [Semantic Versioning](https://semver.org/) to determine version numbers
+for FHIR model releases:
+
+<img src="images/semvar.png" alt="Semantic Versioning Example" height="250"/>
+
+### Examples
+
+1. Create first FHIR model release        -> version `0.1.0`
+2. Create new StructureDefinition(s)      -> version `0.2.0`
+3. Remove a StructureDefinition           -> version `1.0.0`
+4. Change StructureDefinition.id          -> version `2.0.0`
+5. Fix typo in StructureDefinition.title  -> version `2.0.1`
+
+## Release Overview
+
+For Kids First, a release of the FHIR model will consist of a little more than
+the NPM package. Here is an overview:
+
+<p align="center">
+<img src="images/release.png" alt="FHIR Model Release"/>
+</p>
+
+Kids First will use Github releases and tags on a `release` branch to manage
+FHIR model releases.
+
+## Release Process
+
+### Steps
+A new CLI command will be added to produce a release of the FHIR model
+
+```shell
+# Command spec
+$ fhirmodel release <IG control file> <version type>
+
+# Example
+$ fhirmodel release site_root/ig.ini minor
+```
+
+When this command is run the following steps will be executed in order
+to produce a release.
+
+1. **Generate the next release version number**
+
+    - CLI will use the last release's version number and the version type
+    to generate the next version number
+
+2. **Prep conformance resources for release**
+
+    - The `version` attribute on all conformance resources will populated
+      with the new version number.
+    - The `publisher` attribute on all conformance resources with the
+      Kids First canonical URL will be populated with value in
+      `ImplementationGuide.publisher`
+    - The `contact` attribute on all conformance resources with the
+      Kids First canonical URL will be populated with appropriate information
+    - The `date` attribute on all conformance resources will be populated with
+      the current date.
+
+3. **Produce the NPM package for the model**
+
+    - Run the IG publisher to produce the NPM package for the model
+    - Create a tarball of the NPM package
+
+3. **Update `CHANGELOG.md` with release notes**
+
+    - Generate release notes using titles of Git Pull Requests merged into
+      `master` branch since last release
+
+4. **Create a release-version branch and Pull Request**
+
+    - Create a release-version branch off of master: `release-<version number>`
+    - Create a Pull Request to merge this branch into `release` branch
+    - Set the PR description to the release notes markdown
+
+5. **Team must review and approve the Pull Request**
+
```

Comment:
Bullet list the item's the team will check, tests that will run, etc.

Reply:
See new changes